core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -286,9 +285,9 @@ class SearchService(
   def fileRemoved(f: FileObject): Unit = fileChanged(f)
   def fileAdded(f: FileObject): Unit = fileChanged(f)
 
-  def shutdown(): Future[Unit] = {
-    db.shutdown()
-  }
+  def shutdown(): Future[Unit] = Future.sequence {
+    List(db.shutdown(), index.shutdown())
```

Comment:
heh, I actually **just** added a `index.shutdown` in my PR too because I noticed it was missing :smile: